### ISSUE TITLE
Dragonflight support

### ISSUE THREAD

**Issue by Skylli202** [HarielMods/RaidSkipper#1]
Hi,

First, thank you very much for your addon. Really much appreciated and very useful. I would like to know if you would accept PR to support Dragonflight VOTI ? I think it just a matter of adding some data in [data.lua](/Data.lua) file. But as it is also adding a new extension key, maybe not. I've never developed a wow addon in the past, but I would be more than happy to work on the MR !

I've already looked for the VOTI quest on wowhead:
- Normal : [71018](https://www.wowhead.com/quest=71018/vault-of-the-incarnates-break-a-few-eggs)
- Heroic : [71019](https://www.wowhead.com/quest=71019/vault-of-the-incarnates-break-a-few-eggs#comments)
- Mythic : [71020](https://www.wowhead.com/quest=71020/vault-of-the-incarnates-break-a-few-eggs)

> Unlike some other raid skip quest in the past, unlocking the mythic difficulty does not grant skip for lower difficulty. I could not find any in game confirmation.

**Comment by Skylli202** [comment]
MR #1

**Comment by BS5** [comment]
Wow! Thank you for your comment! I fall behind with current content as I am a very casual player that enjoys old content. I just added the new raid ids but I didn't see this thread until now. I will try to find out if Mythic allows Heroic and Normal as well. Thank you again for your comment!

**Comment by BS5** [comment]
I'm reviewing your PR still. You added things that I forgot to! I'll see if I can include parts of your PR or something to include your work.

**Comment by HarielMods** [comment]
I added the instance ids to your PR as comments. Get those changed and I'll approve the PR. Thank you for being my first contributor!

**Comment by Skylli202** [comment]
Thanks a lot for the instanceId, I could not figure out what they were from your previous work. I pushed a new commit with the instanceId you said.

I also added the ATSC normal quest ID as it was missing from my last work where wowhead did not had the quest registered yet.

For the sake of having the intels here : 
- ATSC NM : [link](https://www.wowhead.com/quest=76083/aberrus-the-shadowed-crucible-echoes-of-the-earth-warder)
- ATSC HM : [link](https://www.wowhead.com/quest=76085/aberrus-the-shadowed-crucible-echoes-of-the-earth-warder)
- ATSC M : [link](https://www.wowhead.com/quest=76086/aberrus-the-shadowed-crucible-echoes-of-the-earth-warder)
